monkey/infection_monkey/config.py

```diff
@@ -22,14 +22,17 @@ def from_kv(self, formatted_data):
         for key, value in list(formatted_data.items()):
             if key.startswith('_'):
                 continue
-            if key in ["name", "id", "current_server"]:
+            if key in ["name", "id", "current_server", "max_depth"]:
                 continue
             if self._depth_from_commandline and key == "depth":
+                self.max_depth = value
                 continue
             if hasattr(self, key):
                 setattr(self, key, value)
             else:
                 unknown_items.append(key)
+        if not self.max_depth:
+            self.max_depth = self.depth
```

Comment:
I need to make sure this works with multiple monkey iterations

Reply:
I assume this is #656